src/main/java/edu/greenblitz/robotName/Field.java

```diff
@@ -97,8 +97,8 @@ public static class FieldConstants{
     }
 
     public static class ScoringPositions{
-        public final static Pose2d SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(5,4),new Rotation2d());
-        public final static Pose2d AMP_SCORE_POSITION = new Pose2d(new Translation2d(3,1),new Rotation2d());
-        public final static Pose2d CLIMB_POSITION = new Pose2d(new Translation2d(7,2),new Rotation2d());
+        public final static Pose2d SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(14.140,5.85),new Rotation2d());
```

Comment:
shouldntwesaqve this as arotation2D?
if therearecoliisiton with MoveToPose add a functions for translations